plugins/woocommerce/src/Internal/PushNotifications/DataStores/PushTokensDataStore.php

```
@@ -0,0 +1,397 @@
+	}
+
+	/**
+	 * Deletes a push token.
+	 *
+	 * @param PushToken $push_token An instance of PushToken.
+	 * @param array     $args Not used, enforced by interface.
+	 * @return void
+	 * @throws InvalidArgumentException If the token can't be deleted.
+	 * @throws Exception If the item to delete is not a push token.
+	 */
+	public function delete( &$push_token, $args = array() ) {
```

PHPMD (2.15.0)

Avoid unused parameters such as '$args'. (undefined)

(UnusedFormalParameter)